README.md

```diff
@@ -6,6 +6,10 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
 
 For more info, visit [Instabug.com](https://instabug.com).
 
+### Prerequisites
+
+- Starting from Instabug-Cordova v9, we require the `compileSdkVersion` on Android to be set to a minimum of `29`.
```

Comment:
Can we mention which file needs to be changed + add a snippet showing where the property is in that file?

Reply:
Added 👍